### ISSUE TITLE
My server is glitched/very laggy for no reason (presumably) because of the Wither Storm (pls help if you can)

### ISSUE THREAD

**Issue by ClingooZ** [nonamecrackers2/crackers-wither-storm-mod#525]
**Minecraft version**
The version of Minecraft the mod is running on
1.18.2
**Mod version**
The version of the mod you are using
Also 1.18.2
**Describe the bug**
Me and my friend were playing on our server, and we were waiting for a symbiote to spawn, and we noticed the Wither storm's segment had 6 tractor beams instead of 3 (meaning there was another segment that wasnt meant to be there), after us failing to get the symbiote to spawn (because the area was completely destroyed) we went back to the nether, but we were very laggy (150 ping) and we were playing on a decent server, and it wasn't smoothing out at all like it usual would. Even now as I'm talking if I go onto the server, it eventually just times out. (It didn't do this before, before it would run just fine) Also is there a way I can kill that extra segment? Like a command or something?
**To Reproduce**
1. Go to the Wither storm (phase 6)
2. Wait for awhile 
3. One or more segments gets 6 tractor beams
4. Go back into the nether and you lag a lot
(I don't know if this always happens, but this is what happened when I did it)
**Crash Reports / Logs**
Crash reports / logs hosted on pastebin, gist, etc.

![Screenshot 2022-08-18 115505](https://user-images.githubusercontent.com/111522099/185441610-ff7e0a43-40b3-4e74-b29a-0cc880c672ef.png)

![Screenshot 2022-08-18 120348](https://user-images.githubusercontent.com/111522099/185441862-92d6db3b-e07c-4408-96c0-aaf24c394440.png)
#

**Screenshots**

**Comment by watersofnazareth** [comment]
This is probably due to block clusters being in unloaded chunks

**Comment by ClingooZ** [comment]
@Thefuzz1234555 oohhh is there a way to fix it? (Like a setting or can I go and load those chunks to stop the lag) But the thing is, is that I can't go and render the chunks because the server is so laggy that it would probably time out before I got to the overworld, unless again, there's a command to render those chunks or to delete the block clusters in the unloaded chunks to stop it from lagging since I'm using a server and can probably use the console to type commands in to hopefully stop it.

**Comment by ClingooZ** [comment]
@Thefuzz1234555 Thanks, but I tried this, and it fixed that problem but... my server is still really laggy and in the console on my server it says this. Again, thanks for the help so far, I really appreciate it.

![Screenshot 2022-08-20 003948](https://user-images.githubusercontent.com/111522099/185729219-9fc5ac7f-505c-4e7b-b63d-e6220473559a.png)

![Screenshot 2022-08-20 004907](https://user-images.githubusercontent.com/111522099/185729405-60ccdafd-f818-4836-b614-f1ffb04d3f2c.png) this appeared as well before the screenshot above it

Also, this is how laggy it is --> Server thread/WARN Can't keep up! Is the server overloaded? Running 26138ms or 522 ticks behind (that's what it said on the console) Also my server is running on 8gb so I'm pretty sure it's not supposed to be this laggy.

So, a little update... I saw something new in the log --> Server thread/WARN UUID of added entity already exists: WitherStormSegmentEntity['Wither Storm Segment'/1681, l='ServerLevel[The Wither Storm Server 2]', x=-680.62, y=279.81, z=918.11]. Is there a chance that this extra segment is lagging the server? And if so, is there a command to kill that specific one?

**Comment by nonamecrackers2** [comment]
Segments are duplicating, this is a known issue, and can cause lag: #222. You can fix this temporarily by killing all the segments and reloading the world.

**Comment by ClingooZ** [comment]
@nonamecrackers2 Thanks but what is the command to kill the segments? I looked everywhere to find what the command is but couldn't find it. 
Update: I found the command and typed it in.. and it worked.. but my server is still really laggy and they keep spawning by the group. I can't seem to fix it at all, not even temporarily. 

Update pt 2: Got it fixed temporarily, its fine now, everytime I join I will just have to kill the clusters and the segments.
